### DISCUSSION TITLE
How do I use custom blog structures ? (like Jekyll's "so simple" , etc)

### DISCUSSION THREAD

**Discussion by younes-io** [quarto-dev/quarto-cli#8031] [Q&A]
### Description

It seems that Quarto doesn't offer blog structures / themes (other than picking different colors).. I don't want to spend a lot of time customizing my own theme.. I prefer to pick a community-made theme and use it for my blog, but I can't find any..

**Comment by mcanouil** [comment]
I am not sure what you want here ...

- https://quarto.org/docs/websites/website-listings.html
- https://quarto.org/docs/output-formats/html-themes.html

Take a look at https://quarto.org/docs/gallery/#websites or https://github.com/mcanouil/awesome-quarto#websites-formats

**Reply by younes-io** [reply]
@mcanouil : thank you for your feedback; to be more specific, I'm looking for blog templates that can be amended.. for example, something like the Hugo Apero theme, a boilerplate basically to get started and amend accordingly..

**Reply by mcanouil** [reply]
You'll have to look in the resources I linked.
There is no alternative.

**Reply by younes-io** [reply]
alright, thank you
